src/arbitraries/Arbitrary.ts

```diff
@@ -3,6 +3,7 @@ import {ChainedArbitrary, FilteredArbitrary, MappedArbitrary, NoArbitrary} from
 import {stringify} from './util'
 
 export abstract class Arbitrary<A> {
+  generator: () => number = Math.random
```

Comment:
The generator should be passed to the sample and sampleWithBias functions instead of being part of an arbitrary.